editor/project_converter_3_to_4.cpp

```diff
@@ -1968,11 +1968,14 @@ bool ProjectConverter3To4::test_conversion() {
 	valid = valid & test_conversion_single_additional("(Disconnect(A,B,C) != OK):", "(Disconnect(A,new Callable(B,C)) != OK):", &ProjectConverter3To4::rename_csharp_functions, "custom rename csharp");
 	valid = valid & test_conversion_single_additional("(IsConnected(A,B,C) != OK):", "(IsConnected(A,new Callable(B,C)) != OK):", &ProjectConverter3To4::rename_csharp_functions, "custom rename");
 
-	valid = valid & test_conversion_single_additional("OS.window_fullscreen = Settings.fullscreen", "ProjectSettings.set(\"display/window/size/fullscreen\", Settings.fullscreen)", &ProjectConverter3To4::rename_gdscript_functions, "custom rename");
+	valid = valid & test_conversion_single_additional("OS.window_fullscreen = Settings.fullscreen", "ProjectSettings.set(\\\"display/window/size/fullscreen\\\", Settings.fullscreen)", &ProjectConverter3To4::rename_gdscript_functions, "custom rename");
```

Comment:
The problem happens probably only in builtin scripts, so should be fixed only there - otherwise it will broke normal .gd files(not tested yet)

Reply:
Built-in scripts should be unescaped before conversion, if they aren't already.

Reply:
My bad, I've totally forgotten that scripts are not escaped. Still, I cannot confirm that built-in scripts are unescaped, i. e. I can still observe #63291 in the latest master.

Reply:
What's the difference between `rename_gdscript_functions` and `check_for_rename_gdscript_functions` btw? Their code seems almost identical (except for the return value) and I'm looking for a way to differentiate built-in and external scripts during conversion.

Reply:
`check_for_rename_gdscript_functions` additionally prints exactly in which line change will happen.
At beginning it was very different than `rename_gdscript_functions`, but I unified this one day, so there is a lot of duplicated code, which could be extracted to another function, but when I tried to do it I failed due some strange Regex memory leaks(this shouldn't be too hard to fix)

I think that adding `is_builtin` is good solution, but this probably will be only needed by custom renames that use " inside regex e.g.
```
OS.window_fullscreen = true -> ProjectSettings.set("display/window/size/fullscreen",true)
```
but not with
```
sort_custom( a , b ) -> sort_custom(Callable( a , b ))
```